tests/classification/test_auc.py

```diff
@@ -55,22 +55,29 @@ def sk_auc(x, y, reorder=False):
 
 @pytest.mark.parametrize("x, y", _examples)
 class TestAUC(MetricTester):
-    @pytest.mark.parametrize("ddp", [False])
+    @pytest.mark.parametrize(MetricTesterDDPCases.name_strategy(), MetricTesterDDPCases.cases_strategy())
```

Comment:
DDP apparently fails and therefore was probably excluded in the tests, yet there is no mention in the docs, that DDP is not supported.

Reply:
@Borda @SkafteNicki any idea how to handle this? removing the full ddp/device strategy and opening an issue to fix auc would be my recommendation

Reply:
Its hard for me to remember, but basically AUC needs ordered input to be working and while we are testing on ordered input when we move to `ddp` it somehow gets unordered. I wonder if we should just set the `reorder` flag to `True` whenever `ddp=True`.